src/Mammon.Tests/Actors/LAWorkspaceActorTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Dapr.Actors.Runtime;
+using Mammon.Actors;
+using Mammon.Models.Workflows;
+
+namespace Mammon.Tests.Actors;
+
+public abstract class BaseUnitTests
+{
+    protected static CostCentreRuleEngine GetInstance()
+    {
+        var inMemorySettings = new List<KeyValuePair<string, string>> {
+            new(Consts.CostCentreRuleEngineFilePathConfigKey, "./Services/testCostCentreRules.json")
+        };
+
+        IConfiguration configuration = new ConfigurationBuilder()
+            .AddInMemoryCollection(inMemorySettings!)
+            .Build();
+
+        return new CostCentreRuleEngine(configuration);
+    }
+}
+
+
+[TestClass]
+[TestCategory("UnitTest")]
+public class LAWorkspaceActorTests : BaseUnitTests
```

Comment:
at the moment I would need some convincing re usefulness of these tests - their logic is inherently very simple but setup here is costly along with some pollution and starting to generate considerable setup which will in principle be more costly to update then changed the logic of the actor itself

Reply:
idea/strategy proposed on the call - we run real workflow against mock rules/most cost centre response and mock usage queries.. and then check cost centre and other actor states for expected totals/ elements included etc..

this could be a special build purely for this test (compilation directive etc)/ but the logic itself (other than inputs/outputs to it) is the same as in real execution

this gives us ability to run the workflow once and check all results in on go